avocado/utils/network/interfaces.py

```diff
@@ -189,20 +200,22 @@ def get_hwaddr(self):
         except Exception as ex:
             raise NWException("Failed to get hw address: {}".format(ex))
 
-    def get_link_state(self):
-        """Method used to get the current link state of this interface.
+    def is_link_admin_up(self):
```

Comment:
Again, you are removing a method without a deprecation message. IMO, we don't need to remove "get_link_state()" method. Since that can be used by other methods. So your new method "is_link_admin_up()" could call this method.

Reply:
Another suggestion is to keep the methods here organized in alphabetical order.